services/backends/commandline/buildandrun.go

```diff
@@ -180,19 +181,16 @@ func (b *buildandrun) Stop(workingDir string, getenv func(string) string) ([]byt
 
 	var out []byte
 	if b.Backend.Commands.Stop != "" {
-		cmd, err := commandline.ConstructCommand(workingDir, b.Service.Path, b.Backend.Commands.Stop, getenv)
-		if err != nil {
-			return nil, errors.WithStack(err)
-		}
+        cmdTokens := commandline.TokenizeCommand(b.Backend.Commands.Stop)
+        byteString, err := pipes.RunStrings(cmdTokens...)
```

Comment:
Could we use AssemblePipes to integrate this into ConstructCommand directly?
Would be nice not to have such differing behavior between the different commands.